src/components/pages/GamePlay/GamePlayMain.js

```diff
@@ -68,7 +68,7 @@ function GamePlayMain(props) {
         <Route component={NotFoundPage} />
       </Switch>
 
-      <ChildFooter />
+      {/* <ChildFooter /> */}
```

Comment:
maybe @ashtilawat23 can comment if this is needed?  I thought there was a mention on this just being a minimal footer with basic info as compared to the more robust version for the footers in parent pages.

Reply:
Don't leave commented out code, please